code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -958,5 +958,74 @@
 /datum/movespeed_modifier/careful_driving
 	multiplicative_slowdown = 3
 
+/datum/status_effect/midas_blight
+	id = "midas_blight"
+	alert_type = /atom/movable/screen/alert/status_effect/midas_blight
+	status_type = STATUS_EFFECT_REPLACE
```

Comment:
Do you want replace and not refresh?

Reply:
I wasn't sure how that'd work with it being a variable duration effect. I didn't know if that would just restart the existing timer or use the default duration; -1.